WordPress/gutenberg
# Add TabPanel to document overview replacing fake tabs

## What?


Adds the `TabPanel` to the document overview and removes the fake tab buttons. Makes some adjustments to the `TabPanel` necessary for this to work.

## Why?


Better accessibility/UX.

## How?


See above.

## Testing Instructions





1. Open a post or page.
2. Open the document overview.
3. Notice how there is now a `TabPanel` instead of buttons.

### Testing Instructions for Keyboard


The `TabPanel` is perfectly accessible via the keyboard. Access it by pressing tab, left/right arrow keys will cycle options, and enter to activate.

## Screenshots or screencast
